src/helpers/getPackageVersion.js

```diff
@@ -29,8 +31,8 @@ module.exports.getAngularVersion = getAngularVersion
 const getAngularRuntimeVersion = async function (root) {
   let packagePath
   try {
-    // eslint-disable-next-line n/no-missing-require
-    packagePath = require.resolve('@netlify/angular-runtime/package.json', { paths: [root] })
+    const siteRequire = createRequire(root)
+    packagePath = siteRequire('@netlify/angular-runtime/package.json')
```

Comment:
In practice this was finding the `@netlify/angular-runtime` installed in `<project-dir>/.netlify/plugins/node_modules` despite `paths` being just `['<project-dir'>]`